src/locales/en/common.json

```diff
@@ -1119,6 +1120,12 @@
     "error": {
       "smartWalletNeedToBeActivated": "Smart Wallet needs to be activated to proceed.",
       "unsupportedRequestCallRequestType": "Unsupported Call Request type."
+    },
+    "paragraph":{
+      "undeploy_warning_1": "You are connecting to a wallet that&apos;s undeployed, you will not be able to execute any transaction, go to ",
```

Comment:
The aopostrophe html entity code is literally printed in the UI (check the video)

Reply:
Fixed.